css/elements.css

```diff
@@ -854,20 +863,25 @@ li.menu-search-result-term:before {
 
 
 /* Toolbox */
+.toolbox-container {
+  position: absolute;
+  display: none;
+  padding-bottom: 7px;
+}
+
+.toolbox-container.active {
+  display: inline-block;
+}
+
 .toolbox {
-	position: absolute;
+  position: relative;
```

Comment:
Oof this indentation. Maybe send a PR re-indenting this file and rebase on top?

Reply:
https://github.com/tc39/ecmarkup/pull/278